logger.go

```diff
@@ -0,0 +1,23 @@
+package tui
+
+import (
+	"io/ioutil"
+	"log"
+)
+
+var logger Logger = log.New(ioutil.Discard, "", 0)
+
+// Logger provides a interface for the standard logger.
+type Logger interface {
```

Comment:
Do we need another interface identical to `log.Logger`?

Could this be just the subset that `tui` uses, i.e. `Printf`? There isn't a way to access it outside of the `tui` package, so it doesn't need to support any methods not called by `tui` itself.